.pre-commit-config.yaml

```diff
@@ -99,15 +99,15 @@ repos:
         pass_filenames: false
         files: pythnet/pythnet_sdk
       # Hooks for solana receiver contract
-      - id: cargo-fmt-solana-receiver
+      - id: cargo-fmt-pyth_solana_receiver
         name: Cargo format for solana target chain contract
         language: "rust"
         entry: cargo +nightly-2023-03-01 fmt --manifest-path ./target_chains/solana/Cargo.toml --all -- --config-path rustfmt.toml
         pass_filenames: false
         files: target_chains/solana
-      - id: cargo-clippy-solana-receiver
+      - id: cargo-clippy-pyth_solana_receiver
```

Comment:
please follow the naming convention in this file to use hyphens here not underscores